src/code-templates/services/order-service/start.ts

```diff
@@ -1,6 +1,6 @@
 import { logger } from '@practica/logger';
 import { AppError, errorHandler } from '@practica/error-handling';
-import { startWebServer } from './entry-points/api/server';
+import { startWebServer } from './entry-points-fastify/api/server';
```

Comment:
shouldn't the generated code be without fastify in the entry point to encapsulate the server implementation?

Reply:
I wanted the default code base to be executable 'as is' so as a maintainer you can just rum the code and test things